src/utils/web3.js

```diff
@@ -92,18 +97,20 @@ export const getTokenSmartContract = (web3, address) => {
 
 
 export const getICOParameters = async (web3, address) => {
+  let configFile = config.ICOs;
+  if (process.env.NODE_ENV === 'test') { configFile = testConfig.ICOs; }
```

Comment:
please move this check inside config file. And always in `test` export mocked config